### PR TITLE
data wrapper added for login purpose

### PR DESCRIPTION
**Pull request by anil-ingle**
data wrapper added for login purpose

### CODE REVIEW

File: tenant-app/src/main/java/com/krushidj/tenantapp/entity/DataWrapper.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
+package com.krushidj.tenantapp.entity;
+
+;import lombok.Getter;
```
Comment (anil-webonise) [inline tenant-app/src/main/java/com/krushidj/tenantapp/entity/DataWrapper.java]: ; why

Comment (anil-webonise) [reply tenant-app/src/main/java/com/krushidj/tenantapp/entity/DataWrapper.java]: :?

**Review by anil-webonise — CHANGES_REQUESTED** [review]

File: tenant-app/src/main/java/com/krushidj/tenantapp/entity/DataWrapper.java (hunk 1 above)
Comment (anil-ingle) [reply tenant-app/src/main/java/com/krushidj/tenantapp/entity/DataWrapper.java]: sorry mistake

Comment (anil-ingle) [reply tenant-app/src/main/java/com/krushidj/tenantapp/entity/DataWrapper.java]: will change

**Comment by anil-ingle** [comment]
diff with master

**Pull request closed without merging**